.travis/run-unit-tests.sh

```diff
@@ -59,13 +59,13 @@ else
         # This is a PR build, look at all commits from the target branch
         # to HEAD.
         echo "Checking commits:"
-        git log --format=oneline --abbrev-commit ${TRAVIS_BRANCH}..HEAD
-        changed_paths="$(git diff --name-only ${TRAVIS_BRANCH}..HEAD --)"
+        git log --format=oneline --abbrev-commit "${TRAVIS_BRANCH}..HEAD"
+        changed_paths="$(git diff --name-only "${TRAVIS_BRANCH}..HEAD" --)"
     fi
 
     if [ -n "${changed_paths}" ] ; then
         # Check for changes that would require the C code to be re-tested
-        if (echo ${changed_paths} | grep -qE "daemon/unit_test|database") ; then
+        if (echo "${changed_paths}" | grep -qE "daemon/unit_test|database") ; then
```

Comment:
Can we omit the space preceding ; as we do in other places?

```suggestion
        if (echo "${changed_paths}" | grep -qE "daemon/unit_test|database"); then
```